src/site/stages/build/index.js

```diff
@@ -38,6 +45,50 @@ const rewriteVaDomains = require('./plugins/rewrite-va-domains');
 const updateExternalLinks = require('./plugins/update-external-links');
 const updateRobots = require('./plugins/update-robots');
 
+/**
+ * Immediately copies the Webpack build output to a directory outside of
+ * Metalsmith's build destination, then returns a function for use as a
+ * Metalsmith plugin. This plugin copies the files back to their expected
+ * location.
+ *
+ * This is needed because script/build.sh runs Webpack before the content build,
+ * and Metalsmith's build method removes everything in the destination, which
+ * wipes out the output of the Webpack build.
+ *
+ * This can be removed when we move the content build to a new repository and
+ * this script no longer interacts with the Webpack output at all.
+ */
+function preserveWebpackOutput(metalsmithDestination) {
+  const webpackBuildDirName = 'generated';
+  const tempDir = path.join(__dirname, '../../../../tmp/', webpackBuildDirName);
+  const webpackDir = path.join(metalsmithDestination, webpackBuildDirName);
+
+  const webpackDirExists = fs.existsSync(webpackDir);
+
+  // Immediately move the Webpack output to a new directory
+  if (webpackDirExists) {
+    /* eslint-disable no-console */
```

Comment:
Verifying my sanity with a few console logs. Jenkins is getting a very confusing error where it dies because the directory I'm trying to copy doesn't exist...even though I'm only copying it if it does exist....so...that's fun...